### DISCUSSION TITLE
Refactoring

### DISCUSSION THREAD

**Discussion by WilliamWhite1125** [POLO-SOEN341/POLO-SOEN341_Project_F24#136] [General]
What are you working on?

**Comment by TheWaterisLife** [comment]
Sprint_4/App/app/components/home/TeamSection.js

**Comment by Alvin-11** [comment]
Sprint_4/App/app/create_poll/page.js

**Comment by TheWaterisLife** [comment]
Sprint_4/App/app/create_time/page.js

**Comment by Alvin-11** [comment]
print_4/App/app/test/Signout.test.js

**Comment by TheWaterisLife** [comment]
Sprint_4/App/app/evaluate/[team]/[student]/page.js

**Comment by Ukcl** [comment]
Sprint_4/App/app/components/dashboard/TeamBox.js

**Comment by WilliamWhite1125** [comment]
Sprint_4/App/app/team/[teamname]/[studentid]/page.js

**Comment by Alvin-11** [comment]
Sprint_4/App/app/components/home/ParallaxText.js

**Comment by Ukcl** [comment]
Sprint_4/App/app/components/polls/PollCard.js

**Comment by TheWaterisLife** [comment]
Sprint_4/App/app/login/page.js

**Comment by TheWaterisLife** [comment]
Sprint_4/App/pages/api/teams/teams.js

**Comment by TheWaterisLife** [comment]
Sprint_4/App/pages/api/users/users.js

**Comment by Alvin-11** [comment]
Sprint_4/App/app/components/OfficeHours/OfficeHourSlot.js

**Comment by Ukcl** [comment]
Sprint_4/App/app/test/Signout.test.js
